test/core/gas/MarketOrderOtherOfferList.t.sol

```diff
@@ -234,6 +234,9 @@ contract ExternalMarketOrderOtherOfferList_WithMultipleOffersAtManyTicks is Tick
     mgv.marketOrderByLogPrice(_olKey, _logPrice, 2 ** 96, false);
     gas_();
     (, MgvStructs.LocalPacked local) = mgv.config(_olKey);
-    assertLt(_logPrice, LogPriceLib.fromTick(local.bestTick(), _olKey.tickScale));
+    // In some tests the market order takes all offers, in others not
```

Comment:
Not clear why this is relevant, so perhaps state that `local.bestTick()` only makes sense on a non-empty book. Something like:

```suggestion
    // In some tests the market order takes all offers, in others not. `local.bestTick()` must only be called when the book is non-empty
```